recipes/champ/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "champ" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "2e24d8fea58bf65bb13ec646cbb7404c4fc780ca4c07d5eb03c9aca9cb1609bb" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ce/25/b7cba31f1781a7ddbcafb8ff8e96357ffa98d41704ccf7e19bcfe928649a/champ-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  skip: True  # [win]
+
+
+requirements:
+  build:
+    - pkg-config
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - igraph
+    - setuptools
+
+  run:
+    - python-igraph
+    - matplotlib
+    - numpy >=1.13
+    - scipy >=0.19
+    - pandas
+    - h5py
+    - seaborn
+    - tqdm
+    - louvain
+    - scikit-learn
+    - future
+
+
+test:
+  imports:
+    - igraph
+    - louvain
+
+about:
+  home: https://github.com/wweir827/CHAMP
+  license: MIT
```

```suggestion
  license: MIT
  license_family: MIT
  license_file: LICENSE
```
Ideally the licence file should be included in the PyPI tarball by listing it in the `MANIFEST.in` file. As this isn't currently the case you can add it to the recipe directory instead.

---

added license file to folder with recipe